### ISSUE TITLE
Hyphens vs. em dashes vs. en dashes

### ISSUE THREAD

**Issue by gasyoun** [sanskrit-lexicon/PWG#34]
`tacchrutvā svarabhedena - jagāda Pañcat. 199, 21. ` @funderburkjim 

![fdsfdsfdsfsdsfdfds](https://user-images.githubusercontent.com/80761/115069248-fb98b100-9efb-11eb-95ef-681e215db009.png)

The etext contains the wrong dashes.

![fdsfdsfdfdfdsfdsfdsfsd](https://user-images.githubusercontent.com/80761/115069311-110ddb00-9efc-11eb-8554-46e797377fba.png)

Right now we have `-` where `–` should be. And we leave `—` for places like lists: `— 2) aufzählen: gadita`

**Comment by gasyoun** [comment]
@funderburkjim any chance?

**Comment by Andhrabharati** [comment]
> Right now we have `-` where `–` should be.

I had opted to change the 10k+ cases of ` - ` to ` – ` in the pwg text, as a global correction (except at `<L>38153` and `<L>51648`) in my version.

Probably, Jim also could do the same in the cdsl file. Would you, @funderburkjim ?

**Comment by funderburkjim** [comment]
@Andhrabharati 

5608 matches in 5577 lines for "{#[^#]+-" in buffer

So this accounts for about 5k.  How to identify other '-' that should change to '–'.

Incidentally there do exist '–' in current cdsl:  2747 matches in 447 lines for "–"

**Comment by Andhrabharati** [comment]
No, Jim; the intended change is ' - ' (space, hyphen-space) -> ' — ' (space, em-dash, space) at 10170 places (leaving the 2 cases as mentioned above). **I have shown the en-dash above by mistake.**

The '–' (en-dash) that you mentioned above is indeed in the metrical places.
It is different from the '—' (em-dash) which occurs between the Devanagari text and preceding meaning 'numbers' (roman, alphabetic and greek) etc.

**Comment by funderburkjim** [comment]
Current cdsl for 38153 and 51645:
```
<L>38153<pc>4-0091<k1>nAgadalopama<k2>nAgadalopama
{#nAgadalopama#}¦ (1. {#nAga#} 1.f {#-dala + upamA#}) <lex>n.</lex> N. eines Baumes, {%Xylocarpus Granatum Koen.%},
<ls>RATNAM. 254.</ls>
<LEND>

<L>51648<pc>4-1212<k1>PullAraRyamAhAtmya<k2>PullAraRyamAhAtmya
{#PullAraRyamAhAtmya#}¦ ({#Pulla#} 2.- {#a° + mA°#}) <lex>n.</lex> Titel eines Abschnittes im
<ls>AGNI-P.</ls> <ls>MACK. Coll. I,78.</ls>
<LEND>
```

There is no ' - ' in these.  Perhaps some other change to cdsl needed. @Andhrabharati   - please provide your listing of these two.

**Comment by Andhrabharati** [comment]
My file is having these as
L-38153: `(<hom>1.</hom> {#nAga#} 1,f - {#dala#} + {#upamA#})`
L-51648: `({#Pulla#} 2. - {#a°#} + {#mA°#})`

As I look at these now, it appears that these places should also be changed to the em-dash.

BTW, I have marked the homonyms (`<hom>n</hom>`) throughout the file, as shown above!

Also, it may be seen that I had changed `1. f` to `1,f` as is the practice in the PWG print, making the digital text with "uniform style".
[**There are quite many similar changes that were done in my file!**]

**Comment by funderburkjim** [comment]
@gasyoun   You are probably saying **Finally, my suggestion has been acted on by cdsl!**

Probably this issue may be closed.

**Comment by gasyoun** [comment]
Gentlemen, I give you my thanks. However small, it was important for me, @funderburkjim and 
@Andhrabharati -- love to see how you support each other. It is such a huge amount of work done silently.
